ui/shared/tabs/LogTab.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LazyLog } from 'react-lazylog';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExpand, faFileAlt } from '@fortawesome/free-solid-svg-icons';
+
+import TextLogStepModel from '../../models/textLogStep';
+import { getLogViewerUrl } from '../../helpers/url';
+
+const errorLinesCss = function errorLinesCss(logErrors) {
+  const style = document.createElement('style');
+  const rule = logErrors
+    .map(({ lineNumber }) => `a[id="${lineNumber}"]+span`)
+    .join(',')
+    .concat('{background:#fbe3e3;color:#a94442}');
+
+  style.type = 'text/css';
+  document.getElementsByTagName('head')[0].appendChild(style);
+  style.sheet.insertRule(rule);
+};
+
+class LogTab extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      highlight: null,
+    };
+  }
+
+  componentDidMount() {
+    const {
+      selectedTaskFull: { id: taskId },
+    } = this.props;
+
+    TextLogStepModel.get(taskId).then((textLogSteps) => {
```

Comment:
I know I will need to change this when your PR lands.

Reply:
Since I'm breaking up my pr into two, go ahead and merge this and I'll change it in one of my prs.